### PR TITLE
Don't change css variables

### LINKED ISSUE #86: Issue with CSS variable `--brightest`
Consider the following CSS:

```css
:root {
  --brightest: #fff;
}

body {
  background: var(--brightest);
}
```

The transformed CSS:

```css
:root {
  --bleftest: #fff;
}

body {
  background: var(--brightest);
}
```

I guess this is due to the naive algorithm used for [replacing right/left](https://github.com/MohammadYounes/rtlcss/blob/222a98c3b5fcb75edb77167ce09b2a61c9fbd0ae/lib/plugin.js#L180-L184)?
Does this require adding support for CSS variables? A solution might be to filter `--*` since that is the prefix used for a lot of the proposed extensions to CSS.

### PR DESCRIPTION
**Pull request by danfooo**
When variable names contain direction keywords, without this commit they
get changed.

Fixes #86.

### CODE REVIEW

**Comment by MohammadYounes** [comment]
Thanks @danfooo 

I am not sure if this should be added to the core or not? That's why I left it open for discussion.

Please add a test and we are good to go!

**Comment by danfooo** [comment]
Hi there,

yeah, I think variables should never be touched. Even if they also the call side (where the var is referenced) would be changed, the semantics of the completely free form name would be hard to guess.

I added some tests, I hope they fit the current structure, let me know if now. Thanks!

**Comment by danfooo** [comment]
Hey @MohammadYounes, could you take a look?

File: lib/plugin.js [hunk 1]
```diff
@@ -175,6 +175,13 @@ module.exports = {
     ]
   },
   'processors': [
+    {
+      'name': 'variable',
+      'expr': /^--/im,
+      'action': function (prop, value) {
+        return { prop, value }
```
Comment (MohammadYounes) [inline lib/plugin.js]: @danfooo Please change this to ES5 syntax, just for the sake of code base consistency :)

**Comment by danfooo** [comment]
Ah right. Done ✅🙂
@MohammadYounes

**Comment by MohammadYounes** [comment]
Looks Good 👍 

Thanks!

**Pull request merged**